aspnet/web-forms/overview/getting-started/getting-started-with-aspnet-45-web-forms/membership-and-administration/samples/sample3.cs

```diff
@@ -41,7 +41,7 @@ internal void AddUserAndRole()
         UserName = "canEditUser@wingtiptoys.com",
         Email = "canEditUser@wingtiptoys.com"
       };
-      IdUserResult = userMgr.Create(appUser, "Pa$$word1");
+      IdUserResult = userMgr.Create(appUser, Configuration["AppUserPasswordKey"]);
```

Comment:
Sorry, I was thinking this was ASP.NET Core. You'll need to use the Web Forms configuration approach.